src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -163,6 +163,7 @@ private IList<Location> NormalizeRawMessage(string rawMessage, out string normal
 
```

Comment:
Not shown in this review, but on Line 162, the assignment to `normalizedMessage` isn't necessary. It's assigned at the very end of the message.

Removing this does not introduce a latent error because if in future we add an "early out" path that avoids that assignment, it won't compile.

Reply:
Done